tests/integration/solidity_contracts/PlainOpcodes/test_safe.py

```diff
@@ -1,38 +1,39 @@
 import pytest
 
+from tests.utils.constants import ACCOUNT_BALANCE
+
 
 @pytest.mark.asyncio
-@pytest.mark.Counter
+@pytest.mark.Safe
 @pytest.mark.usefixtures("starknet_snapshot")
 class TestSafe:
     class TestReceive:
-        async def test_should_receive_eth(self, safe, owner, eth):
-            await eth.mint(owner.starknet_address, (int(1e9), 0)).execute(
```

Comment:
no need to mint here, as the account is already funded with (1e9) amount of tokens so that it could pay for its deployment fee.

Hence, the balance of token for `owner` is `1e9 - DEPLOY_FEE`,  which is represented by the constant `ACCOUNT_BALANCE` { this constant is later on used in other places }